---
a1: 1
a2: 1
d3: 0.5
alpha2: 3.1415
theta1: 0
theta2: 0
joint1: {
 rpy: "0 -0 0",
 pos: "0 -0 0",
}
joint2: {
 rpy: "0 -0 0",
 pos: "1 -0 0",
}
joint3: {
 rpy: "3.1415 -0 -0",
 pos: "1 0 0",
}
...